### PR TITLE
RATEST-67: Write Attachments page

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID https://issues.openmrs.org/browse/RATEST-67

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
+	
+	private static final By ATTACHMENTS_LINK = By.cssSelector("i.right:nth-child(1)");
+	private static final By ATTACH_FILE = By.cssSelector(".dz-default");
+	private static final By ADD_CAPTION = By
+	        .xpath("/html/body/div/div[3]/div[3]/div[1]/att-file-upload/div[2]/div/div[2]/textarea");
```
Comment (HerbertYiga) [inline src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java:14]: we can have this at the same line

Comment (kdaud) [reply src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java]: > we can have this at the same line

Have made the recommendation

File: src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
+	
+	private static final By ATTACHMENTS_LINK = By.cssSelector("i.right:nth-child(1)");
+	private static final By ATTACH_FILE = By.cssSelector(".dz-default");
+	private static final By ADD_CAPTION = By.xpath("/html/body/div/div[3]/div[3]/div[1]/att-file-upload/div[2]/div/div[2]/textarea");
```
Comment (kaweesi) [inline src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java:13]: try improving this reference, figure out a more less tree based access to it

File: src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.Scanner;
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
+	
+	private static final By ATTACHMENTS_LINK = By.cssSelector("i.right:nth-child(1)");
+	private static final By ATTACH_FILE = By.cssSelector(".dz-default");
```
Comment (kaweesi) [inline src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java:12]: use className

File: src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java (hunk 2 above)
Comment (kdaud) [reply src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java:13]: > try improving this reference, figure out a more less tree based access to it

This sounds simple. Not sure how but am going to find out.

File: src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java (hunk 3 above)
Comment (kdaud) [reply src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java:12]: Alright

**Pull request closed without merging**